### ISSUE TITLE
sbt new firewall issues

### ISSUE THREAD

**Issue by ekrich** [sbt/sbt#4140]
`sbt new` seems to be gaining usage so we have seen the same problem in the last 2 days. I am just reporting so I am not having this problem.

The first problem occurred on a Mac behind a corporate firewall.
```
sbt new scala-native/scala-native.g8
[info] Loading settings from idea.sbt,plugins.sbt ...
[info] Loading global plugins from /Users/rkuma13/.sbt/1.0/plugins
[info] Set current project to scala-native (in build file:/Users/rkuma13/programs/poc/scala-native/)
[info] Set current project to scala-native (in build file:/Users/rkuma13/programs/poc/scala-native/)
[error] org.eclipse.jgit.api.errors.TransportException: git@github.com:scala-native/scala-native.g8.git: Auth fail
```

The second problem gave the same error as well.
> when i m executing this-sbt new scala/hello.g8.it is showing this error- org.eclipse.jgit.api.errors.TransportException: git@github.com:com scala/hello.g8.git: Auth fail

The first person found that there where not able to `clone` using `git` but `https` worked.
```
git clone git@github.com:scala-native/scala-native.g8.git
Cloning into 'scala-native.g8'...
Warning: Permanently added 'github.com,192.30.253.113' (RSA) to the list of known hosts.
Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```
```
git clone https://github.com/scala-native/scala-native.g8.git
Cloning into 'scala-native.g8'...
remote: Counting objects: 309, done.
remote: Compressing objects: 100% (7/7), done.
remote: Total 309 (delta 0), reused 9 (delta 0), pack-reused 298
Receiving objects: 100% (309/309), 490.05 KiB | 1.40 MiB/s, done.
Resolving deltas: 100% (123/123), done.
```

It seems that `sbt new` defaults to using `git` whereas `https` seems like a more firewall friendly protocol with no special keys needed. This of course could be an entirely wrong conclusion.

sbt version: 0.13.17, 1.1.4

**Comment by eed3si9n** [comment]
I think I fixed that in https://github.com/foundweekends/giter8/pull/313 upstream, but I am guessing that it hasn't reached sbt. See https://github.com/sbt/sbt-giter8-resolver/blob/master/project/Dependencies.scala

**Comment by ekrich** [comment]
Thanks for looking into this issue. Good news that more people are using `sbt new`.

**Comment by eed3si9n** [comment]
This is likely fixed in sbt 1.2.0.
